### PR TITLE
use c_utilities packages

### PR DESCRIPTION
**Pull request by Karsten1987**
connects to ros2/rcl#126

### CODE REVIEW

File: rcl_lifecycle/src/com_interface.c [hunk 1]
```diff
@@ -89,15 +88,28 @@ rcl_lifecycle_com_interface_init(rcl_lifecycle_com_interface_t * com_interface,
...
+  size_t node_name_length = strlen(node_name);
+
+  // Build topic, topic suffix hardcoded for now
+  // and limited in length of 255
+  // e.g. lc_talker__transition_event
+  if ( ((node_name_length + 2 + strlen(pub_transition_event_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_state_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_change_state_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_available_states_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_available_transitions_suffix)) > 255))
```
Comment (wjwwood) [inline rcl_lifecycle/src/com_interface.c]: Why add this now? We're about to have proper checks for this kind of thing. Was it an issue up until now?

Comment (Karsten1987) [inline rcl_lifecycle/src/com_interface.c]: i didn't add it. It was part of the refactoring.
But it's a valid remark, i'll use your validate function.

Comment (wjwwood) [inline rcl_lifecycle/src/com_interface.c]: Sorry I wasn't necessarily suggesting that. You'll have to come back and touch these when all support for namespaces is done. I didn't realize it was already being checked like this.

**Comment by Karsten1987** [comment]
linux: [Build Status](http://ci.ros2.org/job/ci_linux/2487/)
osx: [Build Status](http://ci.ros2.org/job/ci_osx/1945/)
windows: [Build Status](http://ci.ros2.org/job/ci_windows/2598/)

File: rcl_lifecycle/src/com_interface.c (hunk 1 above)
Comment (Karsten1987) [inline rcl_lifecycle/src/com_interface.c]: I'd use the validation for now with the TODO here: https://github.com/ros2/rcl/pull/129/files#diff-adcd6db8c1d9559db68f3cf1dc285d61R56

**Comment by Karsten1987** [comment]
new batch after rebasing:

linux: [Build Status](http://ci.ros2.org/job/ci_linux/2488/)
osx: [Build Status](http://ci.ros2.org/job/ci_osx/1950/)(http://ci.ros2.org/job/ci_windows/2599/)

**Pull request merged**